src/test/cypress/integration/exam/ExamManagement.spec.ts

```diff
@@ -66,7 +66,7 @@ describe('Exam management', () => {
     it('Registers the course students for the exam', () => {
         // We already verified in the previous test that we can navigate here
         cy.visit(`/course-management/${course.id}/exams`);
-        examManagement.getExamRow(examTitle).openStudentRegistration();
+        examManagement.openStudentRegistration(exam.id);
         cy.contains('Registered students: 0').should('be.visible');
```

Comment:
We could reduce the dependency on a translation here by giving the element an id and then asserting that the text contains the number 0. Something like this:
```suggestion
        cy.get('#registered-students-count').should('contain.text', '0').and('be.visible');
```